bundle/print_needed_elf.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# Prints resolved paths to needed libraries for an ELF executable.
+# ldd also does this, but it segfaults in some odd scenarios so we avoid it.
+import sys
+import os
+import subprocess
+from typing import Any, Iterable, List
+
+
+def eprint(msg: Any):
+    print(msg, file=sys.stderr)
+
+
+def run(args: List[str]) -> str:
```

Comment:
add `from __future__ import annotations`, remove the `List` import and then change:
```suggestion
def run(args: list[str]) -> str:
```